### PR TITLE
add max allowed query size

### PR DESCRIPTION
**Pull request by nevermore3**

### CODE REVIEW

File: src/graph/validator/SequentialValidator.cpp [hunk 1]
```diff
@@ -24,6 +25,11 @@ Status SequentialValidator::validateImpl() {
         "given.",
         static_cast<int64_t>(sentence_->kind()));
   }
+  size_t querySize = sentence_->toString().size();
```
Comment (Shylock-Hg) [inline src/graph/validator/SequentialValidator.cpp:28]: I think you could check origin query string.

**Comment by Shylock-Hg** [comment]
Add a tck case

File: src/graph/validator/SequentialValidator.cpp (hunk 1 above)
Comment (nevermore3) [reply src/graph/validator/SequentialValidator.cpp]: done

**Comment by nevermore3** [comment]
> Add a tck case

already has gtest

**Review by CPWstatic — APPROVED** [review]

File: src/graph/validator/test/QueryValidatorTest.cpp [hunk 2]
```diff
@@ -1118,6 +1119,26 @@ TEST_F(QueryValidatorTest, TestMaxAllowedStatements) {
...
+  for (int i = 0; i < count; ++i) {
+    values.append(value);
+  }
+  values.erase(values.size() - 1);
+  query += values;
+  EXPECT_TRUE(checkResult(query));
+  query.append(",\"person_2\":(\"person_2\", 2);");
+  auto result = checkResult(query);
+  EXPECT_FALSE(result);
+  EXPECT_EQ(std::string(result.message()), "SyntaxError: Query is too large (282 > 256).");
```
Comment (Shylock-Hg) [inline src/graph/validator/test/QueryValidatorTest.cpp:1138]: 282 bytes > 256 bytes

**Review by yixinglu — APPROVED** [review]

**Review by jievince — APPROVED** [review]

**Pull request merged**